### PR TITLE
feat: Add Greek sentiment classification

### PR DESCRIPTION
**Pull request by imenelydiaker**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [ ] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Comment by imenelydiaker** [comment]
[The person](https://huggingface.co/sepidmnorozy) behind this dataset created many other datasets covering many dialects and languages, maybe we can copy them all into mteb (one dataset repository with subdirectories) and create one multilingual sentiment classification task?

File: mteb/tasks/Classification/ell/GreekSentimentClassification.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        type="Classification",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["ell-Grek"],
+        main_score="accuracy",
+        date=("2022-08-01", "2022-08-01"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license=None,
```
Comment (isaac-chung) [inline mteb/tasks/Classification/ell/GreekSentimentClassification.py:25]: I've been using "Not specified"

**Review by isaac-chung — APPROVED** [review]
Looks good! I think it'd be a good call to consolidate all these into a multilingual task. Previously I was thinking not to overlap the already covered languages, but I prefer your suggestion for completeness.

File: mteb/tasks/Classification/ell/GreekSentimentClassification.py (hunk 1 above)
Comment (imenelydiaker) [reply mteb/tasks/Classification/ell/GreekSentimentClassification.py:25]: Okay I'll update it 🙂

**Comment by imenelydiaker** [comment]
> Looks good! I think it'd be a good call to consolidate all these into a multilingual task. Previously I was thinking not to overlap the already covered languages, but I prefer your suggestion for completeness.

So we can merge this meanwhile, and I'll open an issue and a PR with this suggestion. Is it good?

**Comment by isaac-chung** [comment]
Sounds good 👍

**Pull request merged**